cpp/daal/src/threading/threading.h

```diff
@@ -43,6 +43,10 @@ typedef void (*functype2)(int i, int n, const void * a);
 typedef void * (*tls_functype)(const void * a);
 typedef void (*tls_reduce_functype)(void * p, const void * a);
 typedef void (*functype_break)(int i, bool & needBreak, const void * a);
+
+typedef int (*loop_functype)(size_t st_idx_reduce, size_t end_idx_reduce, int value_for_reduce, const void * a);
```

Comment:
start_idx_reduce (for consistency)? See loop_functype in cpp/oneapi/dal/detail/threading.hpp